src/librustc_unicode/char.rs

```diff
@@ -362,7 +362,7 @@ impl char {
     /// mostly similar to ID_Start but modified for closure under NFKx.
     #[unstable(feature = "unicode",
                reason = "mainly needed for compiler internals",
-               issue = "0")]
+               issue = "27784")]
```

Comment:
Ok. Seems strange that by writing `#![feature(unicode)]` you then get access to this unrelated/internal thing as well as the possibly-en-route-to-stabilization unicode stuff. But anyway I'll remove this one and squash.